modeltranslation/manager.py

```diff
@@ -541,24 +513,8 @@ def __iter__(self):
         for row in super().__iter__():
             instance.__dict__.update(row)
             for key in self.queryset.translation_fields:
-                if LOOKUP_SEP in key:
```

Comment:
do not remove this part of code because it is one of things why we have this fork